impacket/examples/ntlmrelayx/attacks/httpattacks/adcsattack.py

```diff
@@ -76,7 +76,12 @@ def _run(self):
         certificate = response.read().decode()
 
         certificate_store = self.generate_pfx(key, certificate)
-        LOG.info("Base64 certificate of user %s: \n%s" % (self.username, base64.b64encode(certificate_store).decode()))
+        LOG.info("Writing certificate to %s.pfx" % self.username)
```

Comment:
Why not to add a try except here to catch possible errors?

Reply:
What if file exists already?

Reply:
@666Danger-sudo added try / except block where adcs relay attack will default to original behavior of printing b64 encoded certificate to console if writing to file fails. Also added ability for certificates to be written within lootdir specified in command line